torch/csrc/cuda/Stream.cpp

```diff
@@ -51,6 +60,7 @@ static struct PyMemberDef THCPStream_members[] = {
 };
 
 static PyMethodDef THCPStream_methods[] = {
+  {(char*)"__query__", (PyCFunction)THCPStream_query, METH_NOARGS, nullptr},
```

Comment:
If the name `__query__` appropriate?

Reply:
Or maybe should I remove `query()` in `streams.py` and directly exposing this `_CudaStreamBase` API as `query`?

Reply:
You can add Python docstrings to C functions using `torch._C._add_docstr`. For example, see `torch/_torch_docs.py` which adds documentation to the `torch.xxx` functions.

I think you can do the same thing from `torch/cuda/streams.py` to add documentation to functions on the base C class.

Reply:
Does it mean it is preferred to completely remove `query()` implementation in `torch/cuda/streams.py`?

Reply:
Let me try the following:

1. completely remove `query()` from `torch/cuda/streams.py`.
2. Create a new `torch/cuda/_stream_docs.py` file, and use `_add_docstr` to add `query()` docs
3. import `torch/cuda/_stream_docs.py` in `torch/cuda/__init__.py`

Reply:
I think it's slightly better to put the documentation near the code in `torch/cuda/streams.py` than in a separate `torch/cuda/_stream_docs.py`.

The `_torch_docs.py` is a special case because there are so many torch functions and torch/__init__.py is rather large.